apps/emqx_message_transformation/src/emqx_message_transformation.erl

```diff
@@ -355,9 +358,11 @@ message_to_context(#message{} = Message, Payload, Transformation) ->
         node => node(),
         payload => Payload,
         peername => Peername,
+        pub_props => Props,
         publish_received_at => Message#message.timestamp,
         qos => Message#message.qos,
         retain => emqx_message:get_flag(retain, Message, false),
+        timestamp => Timestamp,
```

Comment:
Maybe it should just duplicate publish_received_at.
or if publish_received_at should be renamed to timestamp ?

Reply:
I was copying the semantics from rule engine: there, `timestamp` is just the time at which processing is happening, for whatever reason...
I also question the usefulness of this field, but Product asked for it: https://emqxio.slack.com/archives/C023T7W5N3Y/p1719909861660009?thread_ts=1718812121.788739&cid=C023T7W5N3Y

Reply:
Replied in DM with internal link to context.